WalletWasabi/Helpers/Constants.cs

```diff
@@ -29,6 +29,8 @@ public static class Constants
 		// There are a maximum of 2,099,999,997,690,000 Bitcoin elements (called satoshis), which are currently most commonly measured in units of 100,000,000 known as BTC. Stated another way, no more than 21 million BTC can ever be created.
 		public const long MaximumNumberOfSatoshis = 2099999997690000;
 
+		public const long MaximumNumberOfWholeBitcoins = 20999999;
```

Comment:
I don't think this is necessary, the UX is not good IMO, and if we should keep it then why we don't round to `20999999.9769` because that is the real maximum.
If someone tries to send more than what they have, they will get a `Not enough coins selected` notification.

Reply:
> why we don't round to 20999999.9769 because that is the real maximum.

I'm too lazy to implement it. UX is good.

Reply:
I will try to do that then.